lib/js/index.ts

```diff
@@ -17,7 +17,9 @@ export { default as Icon } from './components/Icons/Icon';
 export * from './components/Icons/Icon/Icon.consts';
 export { default as IconButton } from './components/Buttons/IconButton';
 export * from './components/Buttons/IconButton/IconButton.consts';
-export { default as Modal } from './components/Modal';
```

Comment:
what about old Modal component? Are we ready to stop exporting it from DS?

Reply:
Reverted, the new one is as `DsModal`

Reply:
Now, it's inconsistent. Why only this component has a prefix?